test/conftest.py

```diff
@@ -7,7 +7,7 @@
 import random
 import unittest
 import uuid
-from test.helpers import create_tables, truncate
+from test.helpers import create_tables, truncate, DB_CONFIG
```

Comment:
This is an optional suggestion. The google style guide recommends not importing specific functions or classes. We as a team tend to relax that a bit for classes and configs but I think for functions it's still a preference. I think the preference here would be to do `from test import helpers` and then add the the dot notations. This is existing code but since we're updating the line it may make sense to bring it within standards.